containers/map-api/routes/pages.js

```diff
@@ -18,7 +18,7 @@ router.post("/add", function(req, res) {
 });
 
 router.get("/", function(req, res) {
-  Pages.find(function(err, pages) {
+  Pages.find({},{},{sort: { page: 1 }},function(err, pages) {
```

Comment:
Does `sort: {page : 1 }` mean that you are start sorting from the first page in the mongodb?

Reply:
Yup. It sorts in ascending order

Reply:
👍